src/data/invalid/Database-mags-invalid.yaml

```diff
@@ -0,0 +1,120 @@
+#invalid b/c img_identifiers is a list
```

Comment:
thanks for the valid and invalid examples!

could we call this file `src/data/invalid/Database-mags-forbidden-img_identifiers-list.yaml` ?